code/src/cljs/sixsq/nuvla/ui/edges/events.cljs

```diff
@@ -124,10 +124,19 @@
                                      status (str " (" status ")"))
                     :content message
                     :type    :error})])
-      {:db (assoc db ::spec/next-heartbeats-offline-edges (zipmap
-                                                            (map :parent resources)
-                                                            (map :next-heartbeat resources))
-                     ::main-spec/loading? false)})))
+      {:db (assoc db ::spec/next-heartbeats-offline-edges (as-> (filter (fn [edge] (= (:online edge) false)) resources) offline-edges
```

Comment:
(= (:online edge) false)) 
 could be: (false? (:online edge))
 
  or a more idiomatic way  (filter (comp false? :online) offline-edges)